File: app/src/main/java/org/apache/fineract/data/models/loan/LoanAccountPage.java
package org.apache.fineract.data.models.loan;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * @author Rajan Maurya
 *         On 09/07/17.
 */

public class LoanAccountPage {

    @SerializedName("elements")
    private List<LoanAccount> loanAccounts;
    private Integer totalPages;
    private Long totalElements;

    public List<LoanAccount> getLoanAccounts() {
        return loanAccounts;
    }

    public void setLoanAccounts(
            List<LoanAccount> loanAccounts) {
        this.loanAccounts = loanAccounts;
    }

    public Integer getTotalPages() {
        return totalPages;
    }

    public void setTotalPages(Integer totalPages) {
        this.totalPages = totalPages;
    }

    public Long getTotalElements() {
        return totalElements;
    }

    public void setTotalElements(Long totalElements) {
        this.totalElements = totalElements;
    }

    @Override
    public String toString() {
        return "LoanAccountPage{" +
                "loanAccounts=" + loanAccounts +
                ", totalPages=" + totalPages +
                ", totalElements=" + totalElements +
                '}';
    }
}

File: app/src/main/java/org/apache/fineract/data/models/loan/LoanAccountPage.kt
package org.apache.fineract.data.models.loan

import com.google.gson.annotations.SerializedName

/**
 * @author Rajan Maurya
 * On 09/07/17.
 */

data class LoanAccountPage(
    @SerializedName("elements") var loanAccounts: List<LoanAccount>? = null,
    @SerializedName("totalPages") var totalPages: Int? = null,
    @SerializedName("totalElements") var totalElements: Long? = null
)